pkg/pillar/nireconciler/linux_config.go

```diff
@@ -806,6 +807,14 @@ func (r *LinuxNIReconciler) getIntendedNIL3Cfg(niID uuid.UUID) dg.Graph {
 			Type:     unix.RTN_UNREACHABLE,
 			Protocol: unix.RTPROT_STATIC,
 		},
+		OutputIf: generic.NetworkIf{
+			// Linux automatically adds "dev lo" to unreachable IPv6 routes.
```

Comment:
Now that we explicitly specify the loopback interface for unreachable IPv6 route, it does not matter what the default behavior for unspecified interface is and whether it changes in future versions. The goal of one of these commits was to avoid having unspecified fields and having Linux set some default values.